taipy/gui/viselements.json

```diff
@@ -899,13 +899,11 @@
                     {
                         "name": "format",
                         "type": "str",
-                        "default_value": "None",
```

Comment:
Quick comment on the properties order (used in the doc).
I would like to see:
value, type, min, max, delta, threshold, show_value, format, delta_format, width, height.

Reply:
resolved